cmake/LLVMIRUtil.cmake

```diff
@@ -111,9 +111,25 @@ function(llvmir_attach_bc_target)
   # compile lang flags
   llvmir_extract_lang_flags(IN_LANG_FLAGS ${LINKER_LANGUAGE})
 
+  # Find all header files
+  foreach(IN_FILE ${IN_FILES})
+    get_filename_component(FILE_EXT ${IN_FILE} LAST_EXT)
+    get_filename_component(FILE_DIR ${IN_FILE} DIRECTORY)
+    if ("${FILE_EXT}" STREQUAL ".h" OR "${FILE_EXT}" STREQUAL ".hpp")
+      # Add to include list
+      list(APPEND INCLUDE_FILES " -I${FILE_DIR}")
```

Comment:
what is this `INCLUDE_FILES` variable? I cannot find it in the documentation.

Reply:
Moreover, I'd suggest adding this to a list and printing it with a warning outside the loop.
Then, go on to use it by adding it to the target property [`INCLUDE_DIRECTORIES`][1] or something along these lines.

[1]: https://cmake.org/cmake/help/v3.3/prop_tgt/INCLUDE_DIRECTORIES.html

Reply:
I think `INCLUDE_FILES` is a mistake. I have fixed that in the newest commit. Thanks for pointing that out!